From 3e88ccff3ba573fb6d1ab746596411eebe310a6b Mon Sep 17 00:00:00 2001
From: John Zimmermann <me@johnnynator.dev>
Date: Thu, 31 Dec 2020 16:24:16 +0100
Subject: [PATCH] Do not match exact amount of spaces for errno macro expansion
 in approvalTests.py

E.g. musl libc expands errno() to __errno_location() without a space between, glibc has 1 space.
---
 scripts/approvalTests.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/approvalTests.py b/scripts/approvalTests.py
index f8b9425c7c..73e9b83239 100755
--- a/scripts/approvalTests.py
+++ b/scripts/approvalTests.py
@@ -44,7 +44,7 @@
 
 # errno macro expands into various names depending on platform, so we need to fix them up as well
 errnoParser = re.compile(r'''
-    \(\*__errno_location\ \(\)\)
+    \(\*__errno_location\s*\(\)\)
     |
     \(\*__error\(\)\)
     |
